tests/bin/test_tmux_layout.py

```
@@ -1,301 +1,301 @@
+        busd.ensure_session()
+        time.sleep(0.5)
+        
+        # PMAI用のディレクトリを作成
+        pmai_dir = tmppath / "work" / "PMAI"
+        pmai_dir.mkdir(parents=True)
+        
+        # PMAIエージェントを起動
+        pane = busd.spawn_child("PMAI", "work/PMAI", "frames/pmai/CLAUDE.md", "Parent Agent")
+        
+        # ペインIDが取得できたか確認
+        assert pane is not None, "Failed to get pane ID for PMAI"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)